decosjoin/tests/fixtures/zaken_response2_part2.json

```diff
@@ -105,6 +105,32 @@
           "rel": "self"
         }
       ]
+    },
+    {
+      "fields": {
+        "company": "Haarlem",
```

Comment:
Huh?

Reply:
Deze snapte ik ook niet, is de achternaam van de aanvrager blijkt. maar dit moet nog verder uitgewerkt worden.